doc/man3/RSA_get0_key.pod

```diff
@@ -82,10 +110,31 @@ needed, duplicate the received value using BN_dup() and pass the
 duplicate.  The same applies to RSA_get0_factors() and RSA_set0_factors()
 as well as RSA_get0_crt_params() and RSA_set0_crt_params().
 
+The size for the return array of RSA_get0_multi_prime_factors() and
+RSA_get0_multi_prime_params() is not given by those two functions.
```

Comment:
"those" -> "these". But I feel that a rewording would be appropriate. I mean what does "to be given" mean in this context?

Reply:
That means: 'you can't get the size of the array by using the who functions'...

Reply:
or 'not returned', more appropriately...

Reply:
'not returned' doesn't really cut it either. What I'm talking about is rather *complete* rewording, not just replacing a word in this sentence. Rewording that would eliminate *any* doubt that size of array is *implied* and is caller's responsibility.

Reply:
Seems the whole paragraph already has the clear meaning even without this sentence, so I decide to remove it...